modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -94,13 +94,21 @@ const INVOICE_EXPIRATION_SECONDS_DEFAULT: u32 = 3600;
 pub enum SendState {
     Funding,
     Funded,
-    Success([u8; 32]),
+    Success,
```

Comment:
Is there any reason we no longer want to return the preimage here? Wouldn't integrating apps potentially want this information?